pkg/kapp/cmd/app/apply_flags.go

```diff
@@ -49,6 +49,7 @@ func (s *ApplyFlags) SetWithDefaults(prefix string, defaults ApplyFlags, cmd *co
 
 	cmd.Flags().StringVar(&s.AddOrUpdateChangeOpts.DefaultUpdateStrategy, prefix+"apply-default-update-strategy",
 		defaults.AddOrUpdateChangeOpts.DefaultUpdateStrategy, "Change default update strategy")
+	cmd.Flags().BoolVar(&s.AddOrUpdateChangeOpts.DryRun, prefix+"dry-run", false, "Execute a server side dry run without persisting changes on the cluster")
```

Comment:
I chose to use `s.AddOrUpdateChangeOpts.DryRun` here as it is being used in `add_or_update_change.go` primarily. Do we have any preferences when it comes to this over:
- ` s.ClusterChangeOpts.AddOrUpdateChangeOpts.DryRun`
- ` s.ClusterChangeOpts.DryRun`
-  `s.DryRun`